### PR TITLE
Feature/graphql client

### PR DESCRIPTION
**Pull request by alienalien13**
🎉

### CODE REVIEW

File: src/graphql/apollo.js [hunk 1]
```diff
@@ -1,5 +1,68 @@
...
+
+const http = new HttpLink({
+  uri: 'https://gql.now.sh/',
+})
+
+const link = from([auth, error, http])
+
+const client = new ApolloClient({
+  link,
+  cache: new InMemoryCache(),
```
Comment (flashlee) [inline src/graphql/apollo.js]: Do we really need explicitly define this? apollo-boost readme says it does these things by default. https://github.com/apollographql/apollo-client/blob/master/packages/apollo-boost/README.md
`link` can be set in `uri` option and `cache` has default value `InMemoryCache` from `apollo-cache-inmemory` [Link](https://www.apollographql.com/docs/react/essentials/get-started.html#configuration)

File: package.json [hunk 2]
```diff
@@ -5,9 +5,14 @@
   "author": "illWatchers",
   "license": "GPL-3.0-or-later",
   "dependencies": {
-    "apollo-boost": "0.1.16",
+    "apollo-boost": "0.1.17",
+    "apollo-cache-inmemory": "1.3.6",
+    "apollo-link": "1.2.3",
+    "apollo-link-error": "1.1.1",
+    "apollo-link-http": "1.5.5",
```
Comment (flashlee) [inline package.json]: apollo-boost [already includes these packages](https://github.com/apollographql/apollo-client/blob/master/packages/apollo-boost/README.md). I assume that if we want to have more control over configuration let's don't include `apollo-boost` into our dependencies at all. [Link for further reading](https://www.apollographql.com/docs/react/advanced/boost-migration.html)

Comment (alienalien13) [reply package.json]: @flashlee good catch, will adjust it 👍

**Comment by alienalien13** [comment]
Can't export onError from graphql-boost, will try other ways

File: src/graphql/apollo.js (hunk 1 above)
Comment (alienalien13) [reply src/graphql/apollo.js]: Let me check it later

**Comment by flashlee** [comment]
> Can't export onError from graphql-boost, will try other ways

I think in case we are using `ApolloClient` from apollo-boost there's no requirement to import `onError` explicitly. Apollo-boost provides own default error handler. If we want to use custom handler we just need to add `onError` option to `ApolloClient` with our handler.

**Review by maybelinot — APPROVED** [review]
approffed

**Pull request merged**